packages/components/NavBar.tsx

```diff
@@ -53,7 +53,7 @@ function NavBar() {
   );
 
   return (
-    <Box position="fixed" w="100%" top={0} shadow="base" zIndex={1} bg={navbarBgColor}>
+    <div className={`navbar fixed p-0 shadow-md ${navbarBgColor}`}>
```

Comment:
Do we have somethig like [`clsx`](https://www.npmjs.com/package/clsx) or `cn`?

Not really hard to have a basic implementation of a util that does this either, if we don't want to install a new dep (even tho I personally think it's a good ideia, might be a @marco-souza 's call).

The problem of using template literals for dynamic classNames is that it doesn't sanitize falsy values, for instance. This leads to classNames like `mt-4 border-full undefined` or `mt-4 border-full false`.

Reply:
For reference: https://bundlephobia.com/package/clsx@2.1.1

Reply:
Looking at our list of dependencies, we don’t have any that handle ensuring the correct class names.

I wasn’t aware of this issue, and like you, I think it’s important to address it—whether we implement the solution ourselves or use the library you mentioned.

Thanks for bringing this up for discussion.

Let’s wait for the other contributors to weigh in so we can decide how to approach the topic, but in my opinion, we could use the clsx library.

Reply:
I don't think we need a new dependency for this, as the following code does the same - or almost
- https://github.com/podcodar/app/blob/main/src/shared/tw.tsx

```ts
export function classes(...classes: Optional<string>[]) {
  return classes.filter(Boolean).join(" ");
}
```

I'd also add tests for common usage scenarios for this `classes`

Reply:
Made in: https://github.com/podcodar/webapp/pull/175/commits/14479cd3015d421b06724fceaa97fea232d90e22